src/Middleware/WebSockets/src/ServerWebSocket.cs

```diff
@@ -0,0 +1,80 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Net.WebSockets;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.WebSockets;
```

Comment:
I don't think we used file-level namespace statement (name?) in 6.0.

Reply:
Fixed. Thanks.